libc/spec/posix.td

```diff
@@ -1618,6 +1618,17 @@ def POSIX : StandardSpec<"POSIX"> {
                 ArgSpec<VoidPtr>
             ]
         >,
+        FunctionSpec<
+            "lfind",
+            RetValSpec<VoidPtr>,
+            [
+                ArgSpec<VoidPtr>,
+                ArgSpec<VoidPtr>,
+                ArgSpec<SizeTPtr>,
+                ArgSpec<SizeTType>,
+                // TODO: Unsure how to specify int(*compar)(void *, void *)
```

Comment:
You'd have to declare a type for the function pointer first, then use that as an ArgSpec.   If you look at how other functions that have function pointer parameters, such as bsearch or qsort, those should provide an example.

Reply:
Resolved by [e1f3019](https://github.com/llvm/llvm-project/pull/114692/commits/e1f301948492ced96cc5428ebe91e7c18f791d62)